utils/build-apple-framework.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Does this script really require `bash` instead of just `sh`?

Reply:
I guess it might be for features such as testing if a command exists? It’s no biggie, every macOS installation should support this anyways.

Reply:
This script uses the `function` keyword, `local`, `[[ ]]` and arrays, all of which are bash extensions

Reply:
I should rephrase; I meant more, are those features all needed? But again, it actually doesn’t really matter, all macOS installations will have the dependency.

Reply:
That's a good question. I started off with the script from React Native and left that header.

Reply:
Okeydokey